packages/cli/src/commands/versions/bump.ts

```diff
@@ -141,9 +164,42 @@ export default async () => {
     });
 
     console.log();
-    console.log("Running 'yarn install' to install new versions");
+    console.log(
+      `Running ${chalk.blue('yarn install')} to install new versions`,
+    );
     console.log();
     await run('yarn', ['install']);
+
+    if (breakingUpdates.size > 0) {
+      console.log();
+      console.log(
+        chalk.yellow('The following packages may have breaking changes:'),
+      );
+      console.log();
+
+      for (const [name] of breakingUpdates) {
```

Comment:
Based on your screen shot, it'd be awesome if `breakingUpdates` was sorted alphabetically.  e.g., it always seems to make most sense to me to see related plugins together, esp say if there's plugin-vendor and plugin-vendor-backend.  I note it because your screen shot seemed to be somewhat random based on how it populated the list?